src/main/scala/beam/agentsim/infrastructure/parking/ParkingZone.scala

```diff
@@ -119,7 +125,6 @@ object ParkingZone extends LazyLogging {
       geoId,
       parkingType,
       VehicleManager.AnyManager,
-      Some(SitePowerManager.createId(DefaultParkingZoneId.toString)),
```

Comment:
PowerManagerSpec.scala#L45, SitePowerManager.scala#L95 and ZonalParkingManagerSpec.scala#L624 have to be modified to reflect this change.

Reply:
Done